test/e2e/kubectl/kubectl.go

```diff
@@ -450,7 +451,26 @@ var _ = SIGDescribe("Kubectl client", func() {
 			}
 
 			apiGroups, err := c.Discovery().ServerPreferredResources()
-			framework.ExpectNoError(err)
+
+			if discovery.IsGroupDiscoveryFailedError(err) {
+				discoveryErr := err.(*discovery.ErrGroupDiscoveryFailed)
+				for gv := range discoveryErr.Groups {
+					if !strings.HasSuffix(gv.Group, ".k8s.io") {
+						// tolerate discovery errors for non-k8s.io groups (like aggregated/crd groups)
```

Comment:
Would this tolerate discovery errors for groups without the k8s.io suffix like core here?

Reply:
yeah... though those groups are never aggregated, so the likelihood of a discovery error is almost none... but I'll update